examples/mixture_models/dirichlet_mixture_of_multinomials.ipynb

```diff
@@ -65,8 +65,9 @@
    },
```

Comment:
<p>Use coords. This example tries to be more down-to-earth even if using simulated data so the number of clusters (k again) corresponds to the tree species, so it would be great to use as coord values real tree names :). For the number of observations, here they now correspond to the number of forests, so again, as there are only 10 "forests" we could also use real forest names.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/212/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>ah got it, that's a cool idea. will add that!</p><p> </p>